cmd/adsysd/integration_tests/adsysctl_policy_test.go

```diff
@@ -871,6 +901,27 @@ func TestPolicyUpdate(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		// Krb5 library error cases
+		"Error when libkrb5 ccache not present on disk": {
+			initState:          "localhost-uptodate",
+			noExportKrb5cc:     true,
+			detectCachedTicket: true,
+			krb5MockBehavior:   "return_ccache:%s/not_present",
```

Comment:
so, here, you are using a special name on the mock and not an extra argument as in the other test. Maybe you should follow the same convention in the package tests?